tests/test_editor.py

```diff
@@ -47,6 +47,50 @@ def testPositionerWidget(self):
         widget.move_joints_button.click()
         self.assertEqual(self.view.instrument.positioning_stack.set_points[0], 50)
 
+    def testFindInText(self):
+        # Testing search works, and only finds one occurrence
+        window = self.view
+        window.editor.setText("The Text")
+        widget = FindWidget(window)
+        widget.search_box.setText("text")
+        widget.search()
+        self.assertTrue(window.editor.selectedText() == "Text")
```

Comment:
Use self.assertEqual instead

Reply:
Sorry I missed this earlier. Please replace `self.assertFalse(window.editor.selectedText() == "Text")` with `self.assertNotEqual()` and `self.assertTrue(window.editor.selectedText() == "Text")` with `self.assertEqual()`